### DISCUSSION TITLE
🔗 🚀 Feature Request: Allow to select multiple taxonomies in the same Taxonomy Filter (Related to #23418)

### DISCUSSION THREAD

**Discussion by Alexandero89** [elementor/elementor#23341] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When i use the new Taxonomy Filter Widget i can only choose one of the filters at the same time.
But it would be useful to choose multiple taxonomies at the same time.

Example:
I want to see all concerts and book readings.

Situation now:
I filter by "concerts" and only get shown all concerts.
After finishing i have to filter by "book readings" and only get shown all book readings.

### Describe the solution you'd like

The new Taxonomy Filter Widget should have a setting to allow to use multiple Taxonomies.

I should be able to click on the taxonomy "concert" and "book readings" and both type of events should be shown in the list.

### Additional context

I checked the network request that is send by clicking on one of the taxonomy elements.
It looks something like this:
`{"post_id":245,"widget_filters":{"taxonomy":{"product_cat":["concert"]}},"widget_id":"4b13f7d","pagination_base_url":"https://foo.bar/test/"}`

After adding a second category to the "product_cat" list i got the correct result from the server.
Used json:
`{"post_id":245,"widget_filters":{"taxonomy":{"product_cat":["concert","bookreading"]}},"widget_id":"4b13f7d","pagination_base_url":"https://foo.bar/test/"}`
So its only a problem of the frontend it seems.

**Comment by nicholaszein** [comment]
Hello @Alexandero89,

Thanks for contributing your suggestion! 🙏

Your ideas and input as a Web Creator are valuable to us. We're always keen to explore ways to enhance Elementor to meet our community's needs better.

### 💡 Due to the high volume of feature requests we receive and our commitment to continuous improvement, we can't guarantee that every suggestion will be immediately included in our roadmap.

That being said, your idea is important to us. We'll keep your suggestion on our radar, and if it attracts a significant number of votes in the future, we'll consider implementing it.

We appreciate your patience as we take the time to review your request thoroughly. We aim to ensure all decisions align with our mission to provide Web Creators with the tools they need to build their dream websites.

Kind regards

**Comment by nicholaszein** [comment]
Related to:
- #23418
